test/passing/apply.ml

```diff
@@ -51,7 +51,7 @@ let whatever_labelled a_function_name long_list_one some_other_thing =
 
 let cartesian_product' long_list_one long_list_two =
   List.concat
-    (long_list_one
+    ( long_list_one
```

Comment:
The space should have been there since `indicate_multiline_delimiters = closing-on-separate-line`.